docs/general/fixtures.md

```diff
@@ -0,0 +1,255 @@
+# Fixtures
+
+## Intro
+
+NOTE: This is a living document, renames still have to be done.
+
+You could obtain `SIA_TOKEN` from your `Local Storage` after you logged in into `SIA` with your `ADW account`.
+
+Set `$SIA_TOKEN`:
+
+    SIA_TOKEN=YOUR_SIA_TOKEN_FROM_LOCAL_STORAGE:
+
+You could also use the Chrome console to copy the `accessToken`:
+
+    copy(localStorage.getItem('accessToken'))
+
+Backend OpenAPI documentation:
+
+    https://acc.api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/
+
+## Proposals
+
+### 'Consistent' import naming
+
+This is kind of hard with those long file names...
+
+import privateRolesFixture from 'get-signals-v1-private-roles.json'
+import inputRolesSelectorFixture form  selector-models-categories-selectors_inputRolesSelector.json
+
+## Notes
+
+Tests to fix:
+
+  src/signals/settings/departments/Detail/components/CategoryLists/__tests__/CategoryLists.test.js
+  src/signals/settings/departments/Detail/components/__tests__/mapCategories.test.js
+
+Commands:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/' "Authorization:Bearer $SIA_TOKEN" | jq '.results[]  | select(.id == 2)'
+    jq '.categories' department.json
+    jq '.categories | length' department.json
+    jq '.results[] | select(.id == 2)' departments.json
+
+## HTTP Responses
+
+### get-signals-v1-private-roles.json
+
+Old Name: roles.json
+
+Endpoint: signals/v1/private/roles/
+
+Key: results
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/roles/' "Authorization:Bearer $SIA_TOKEN" | jq '.results' > get-signals-v1-private-roles.json
+
+### get-signals-v1-private-categories.json
+
+Old Name: categories_private.json
+
+Endpoint: signals/v1/private/categories/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/categories/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-categories.json
+
+### get-signals-v1-private-departments.json
+
+Old Name: departments.json
+
+Endpoint: signals/v1/private/departments/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/departments/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-departments.json
+
+### get-signals-v1-private-users.json
+
+Old Name: users.json
+
+Endpoint: v1/private/users/
+
+Example:
+
+    http 'https://acc.api.data.amsterdam.nl/signals/v1/private/users/' "Authorization:Bearer $SIA_TOKEN" | jq > get-signals-v1-private-users.json
+
+### get-signals-v1-private-users-id.json
+
+Notes: This fixture is pretty fragile since it needs a specific set of permissions the issue is when
+more then the required permissions are added (different type permissions) tests break badly
```

Comment:
I don't think the fixture persé is fragile; it's the implementation and dependency on the fixture's data that makes it brittle. I would leave this remark out, because we would have to remove or change this line whenever we fix the tests that rely on this particular fixture.
For good measure, we good add user fixtures for specific roles so that, when necessary, we have data that contains certain permissions.

Reply:
I will leave it out then, please keep in mind this document was meant as my personal notes.

Reply:
After thinking about this, when I leave it out I will maybe forget what I meant (to myself) with this remark (and it took me with a previous ticket `way too much time` to figure out what went wrong), let's discuss this somewhere this week, and I will remove it after that.